### DISCUSSION TITLE
好不稳定呀，一直部署失败

### DISCUSSION THREAD

**Discussion by xupipi1125** [zeabur/zeabur#88] [Q&A]
![image](https://user-images.githubusercontent.com/109848250/234255492-cdcfb200-dcc1-4ba4-b525-b9e948f0173d.png)

**Comment by MichaelYuhe** [comment]
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/63531512/234261374-1438ee03-950c-4e4c-bca5-c21361dbb109.png">
本地构建也失败了，您可能需要查看下代码或者 notion 设置，有问题可以加入 Discord 群组联系我们活重新打开讨论～
